test/crawler_web/features/home_page/filter_keyword_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule CrawlerWeb.Features.HomePage.FilterKeywordTest do
+  use CrawlerWeb.FeatureCase, async: false
+
+  feature "view list of uploaded keywords table when filtering by a valid keyword", %{
+    session: session
+  } do
+    user = insert(:user)
+    insert(:keyword, user_id: user.id, name: "buy phone")
+    insert(:keyword, user_id: user.id, name: "buy tv")
+    insert(:keyword, user_id: user.id, name: "watch")
+
+    session
+    |> log_in_user(user)
+    |> visit(Routes.home_path(CrawlerWeb.Endpoint, :index, %{keyword: "buy"}))
```

Comment:
This is a case when the user accesses the page with the `keyword=buy` param

We should add the test case for

1/ User login
2/ Visit the page without the keyword param
3/ Enter the keyword in the search field
4/ Click into the Search button
5/ Assert the result


Another issue:

- It's better if we could assert the `search field has the keyword prefill` when we filter the keyword param.

```
assert_has(_search_field_has_text('buy'))
```